ros/angel_system_nodes/angel_system_nodes/uho_activity_detector.py

```diff
@@ -0,0 +1,341 @@
+# TODO: This is added so that the the angel_system/uho/src folder
+# is found when torch attempts to load the saved model checkpoints.
+# Is there a better way to do this?
+import sys
+sys.path.append("angel_system/uho")
```

Comment:
This seems to only be required because of the additional content in the epoch checkpoint outside of the state dict.